packages/fetchai/connections/p2p_libp2p/aea/api.go

```diff
@@ -51,6 +51,15 @@ type Pipe interface {
 	Close() error
 }
 
+// Needed to break import cycle
+type AgentRecord struct {
```

Comment:
For now, I am using this record only in `aea`. Elsewhere I am using the protobuf message directly